### ISSUE TITLE
Dash to Dock is preventing me from moving the mouse to my secondary monitor

### ISSUE THREAD

**Issue by markscamilleri** [micheleg/dash-to-dock#290]
I have a setup where my secondary/external monitor is to my left. I can't move my mouse further away than the screen's edge, thus I can't move my mouse to the secondary monitor. Both my dock and my external monitor are set to my left.

This does not happen when the applications menu is open.

**Comment by micheleg** [comment]
That a limitation of the current barrier implementation. The expected behaviour however is that the dock is shown and afterwards the mouse pointer should be allowed to move to the secondary screen (See #74). If this is not the behaviour you are experiencing there might be a bug. What are your GNOME Shell and extension versions?

As a workaround I suggest you to either:
* disable intellihide
* disable the "push to show" option in the intellihide settings

I haven't had time to look into nor test the implementation of this related project: https://github.com/passingthru67/workspaces-to-dock/issues/23#issuecomment-176902067 [https://github.com/passingthru67/workspaces-to-dock/commit/518a15171392dd848032874e30c5236534c68bed, https://github.com/passingthru67/workspaces-to-dock/commit/dbb1162038a8ddc1fc405c85c17eb5a4c79ecf4f], which could be a better solution for this problem.

**Comment by markscamilleri** [comment]
My GNOME Shell version is 3.18.3 and the extension version is version 50.

I have found that lowering the pressure threshold from 100 does help and the expected behaviour is more visible and the workarounds you suggested worked. The expected behaviour is understandable. However I've opted to use the first of your suggestions.

**Comment by micheleg** [comment]
Yes, the optimal threshold value depends on your input device and hand sensitivity. Reopening to not forget about it.

**Comment by markscamilleri** [comment]
Ah ok then. Thanks
